root@raspberrypi:/ros2_ws/src/so_arm_100_hardware/config# cat calibration.yaml
timestamp: '2025-09-20T22:45:37.047815'
joints:
  Shoulder_Rotation:
    min:
      ticks: 984
      speed: -50
      load: 0
    center:
      ticks: 2241
      speed: -1
      load: 0
    max:
      ticks: 3651
      speed: -1
      load: 0
  Shoulder_Pitch:
    min:
      ticks: 807
      speed: 0
      load: 0
    center:
      ticks: 1969
      speed: 0
      load: 0
    max:
      ticks: 3154
      speed: 0
      load: 0
  Elbow:
    min:
      ticks: 3949
      speed: 0
      load: 0
    center:
      ticks: 1011
      speed: 50
      load: 0
    max:
      ticks: 1991
      speed: -1
      load: 0
  Wrist_Pitch:
    min:
      ticks: 912
      speed: 0
      load: 0
    center:
      ticks: 3221
      speed: 0
      load: 0
    max:
      ticks: 3221
      speed: 0
      load: 0
  Wrist_Roll:
    min:
      ticks: 3315
      speed: 0
      load: 0
    center:
      ticks: 1113
      speed: 0
      load: 0
    max:
      ticks: 3003
      speed: 0
      load: 0
  Gripper:
    min:
      ticks: 918
      speed: 0
      load: 0
    center:
      ticks: 1405
      speed: 2364
      load: 0
    max:
      ticks: 2118
      speed: 0
      load: 0